### PR TITLE
Add CHANGES.md

### PR DESCRIPTION
**Pull request by arskama**
CHANGES.md explains the changes made to the API since the last Origin Trial

Fixes: #199

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# API Surface Changes
+
+This file describes API surface changes made during experimentation.
+A first Origin Trial was carried out between [M92-M94](https://chromestatus.com/feature/5597608644968448).
```
Comment (tomayac) [inline CHANGES.md:4]:
```suggestion
A first Origin Trial was carried out between [M92–M94](https://chromestatus.com/feature/5597608644968448).
```

File: CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+No real feedback was shared by Google with other parties, incl. Intel
+At the beginning of 2022, the ownership of the Compute Pressure API was transferred to Intel.
+
+## Changes since last Origin Trial
+After research and discussions with interested parties (e.g., Google, Zoom and others), we
```
Comment (tomayac) [inline CHANGES.md:9]:
```suggestion

After research and discussions with interested parties (e.g., Google, Zoom and others), we
```

File: CHANGES.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+decided to make the following major changes to the API shape:
+
+- For better ergonomic, security and fingerprinting [concerns](https://github.com/w3c/compute-pressure/issues/24), the API [interfaces](https://www.w3.org/TR/compute-pressure/#the-pressurerecord-interface) has been redesigned
```
Comment (tomayac) [inline CHANGES.md:12]:
```suggestion
- For better ergonomics, and due to security and fingerprinting [concerns](https://github.com/w3c/compute-pressure/issues/24), the API [interfaces](https://www.w3.org/TR/compute-pressure/#the-pressurerecord-interface) has been redesigned.
```

File: CHANGES.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- Alignment with existing APIs: The Observer pattern used by the specification now more closely [follows](https://github.com/w3c/compute-pressure/issues/21) existing Observer based APIs on the web platform.
+
+- Partner requests: The APIs now works in iframes as well as [workers](https://github.com/w3c/compute-pressure/issues/15) (shared and dedicated) with proper security and privacy mitigations in place.
```
Comment (tomayac) [inline CHANGES.md:16]:
```suggestion
- Partner requests: The APIs now work in iframes as well as [workers](https://github.com/w3c/compute-pressure/issues/15) (shared and dedicated) with proper security and privacy mitigations in place.
```

**Review by tomayac — COMMENTED** [review]
I think to be more useful, this document should outline a _before_ and _after_ state, so developers can more easily see what _actually_ was changed. A good example is this [`CHANGES.md` document](https://github.com/w3c/window-placement/blob/main/CHANGES.md).

**Comment by arskama** [comment]
> I think to be more useful, this document should outline a _before_ and _after_ state, so developers can more easily see what _actually_ was changed. A good example is this [`CHANGES.md` document](https://github.com/w3c/window-placement/blob/main/CHANGES.md).

I will definetly update the document to take this comment into account.
Thanks for reviewing!

File: CHANGES.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+A first Origin Trial was carried out between [M92–M94](https://chromestatus.com/feature/5597608644968448).
+No real feedback was shared by Google with other parties, incl. Intel.
+At the beginning of 2022, the ownership of the Compute Pressure API was transferred to Intel.
+
+## Changes since last Origin Trial
+
+After research and discussions with interested parties (e.g., Google, Zoom and others), we
+decided to make the following major changes to the API shape:
+
+- For better ergonomics, and due to security and fingerprinting [concerns](https://github.com/w3c/compute-pressure/issues/24), the API [interfaces](https://www.w3.org/TR/compute-pressure/#the-pressurerecord-interface) has been redesigned.
```
Comment (kenchris) [inline CHANGES.md:13]: has been redesigned to not use developer-configurable buckets but instead output pressure changes as high level (link to https://github.com/w3c/compute-pressure/blob/main/high-level-states.md)

Comment (arskama) [reply CHANGES.md]: Fixed

**Review by kenchris — APPROVED** [review]

**Comment by arskama** [comment]
@tomayac , the PR was merged, let us know if you have still comments. I ll update the document.

**Pull request merged**